Frame has --json flag for JSON output support (#116)
Intelligence-AI/jsonschema: docs/frame.markdown

```sh
jsonschema frame path/to/my/schema.json
```

> ### Frame a JSON Schema with output in the form of JSON

Reviewer: ```suggestion
### Frame a JSON Schema and output result as a JSON document
```
